chat.js

```diff
@@ -896,7 +896,7 @@ Chat.loadCommands = function () {
 		Object.assign(commands, require('./spacialgaze-plugins/' + file).commands);
 	}
 	for (let file of fs.readdirSync(path.resolve(__dirname, 'game-cards'))) {
-		if (file.substr(-3) !== '.js') continue;
+		if (file.substr(-3) !== '.js' || file === 'SGgame.js') continue;
```

Comment:
If you skip `SGgame.js`, then all the commands from that file get ignored. Also the issue will probably crop up again with other files added to this directory but id have to test to confirm that